pluto_build/version.env

```diff
@@ -1,2 +1,20 @@
-VERSION=22v3
-VERSION_PREV=22v2
+# PLUTO versions for latest release
+VERSION=22v3.1
+VERSION_PREV=22v3
+
+# input data versions for latests minor release
```

Comment:
For consistency purposes, I like that we added all the version_variables for each dataset